src/Arcus.Scripting.Storage.Table/Scripts/Set-AzTableStorageEntities.ps1

```diff
@@ -0,0 +1,93 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of resource group is required"),
+    [Parameter(Mandatory = $true)][string] $StorageAccountName = $(throw "Name of Azure storage account is required"),
+    [Parameter(Mandatory = $true)][string] $TableName = $(throw "Name of Azure table is required"),
+    [Parameter(Mandatory = $true)][string] $ConfigurationFile = $(throw "Path to the configuration file is required")
+)
+
+if (-not (Test-Path -Path $ConfigurationFile)) {
+    throw "Cannot re-create entities based on JSON configuration file because no file was found at: '$ConfigurationFile'"
+}
+if ((Get-Content -Path $ConfigurationFile -Raw) -eq $null) {
+     throw "Cannot re-create entities based on JSON configuration file because the file is empty."
+}
+
+$schema = @'
+{
+  "definitions": {},
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "http://example.com/root.json",
+  "type": "array",
+  "title": "The configuration JSON schema",
+  "items": [{
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "anyOf": [{
+              "type": "string"
+            }, {
+              "type": "null"
+            }
+          ]
+        }
+      },
+      "additionalProperties": false
+    }
+  ]
+}
+'@
+
+if (-not (Get-Content -Path $ConfigurationFile -Raw | Test-Json -Schema $schema -ErrorAction SilentlyContinue)) {
+    throw "Cannot re-create entities based on JSON configuration file because the file does not contain a valid JSON configuration file."
+}
```

Comment:
Great stuff! 🎉
Maybe an idea to add the link to the feature docs?
https://scripting.arcus-azure.net/Features/powershell/azure-storage/azure-storage-table

Reply:
done in [`01058af` (#222)](https://github.com/arcus-azure/arcus.scripting/pull/222/commits/01058af5846c7e44f8f1128dfdeb11fdea71bd1c)